docs/elements.md

```diff
@@ -1994,6 +1995,51 @@ The `pl-card` attributes mirror the options of [Bootstrap 4 cards](https://getbo
 
 ---
 
+### `pl-template` element
+
+Displays boilerplate HTML from templates in a reusable way.
+
+#### Sample element
+
+```html
+<pl-template file-name="outerTemplate.mustache" subdirectory="templates">
+  <pl-variable name="is-open">True</pl-variable>
+  <pl-variable
+    name="problem-statement"
+    directory="question"
+    file-name="serverFilesQuestion/statement.html"
```

Comment:
Looking at the docs below, this is confusing. Shouldn't the directory name `serverfilesQuestion` _not_ be in the `file-name`, since the directory should be specified in `directory`?